doc/source/actors.rst

```diff
@@ -47,22 +78,42 @@ When the above actor is instantiated, the following events happen.
 Actor Methods
 -------------
 
-Any method of the actor can return multiple object refs with the ``ray.method`` decorator:
+.. tabs::
+  .. group-tab:: Python
 
-.. code-block:: python
+    Any method of the actor can return multiple object refs with the ``ray.method`` decorator:
```

Comment:
Here, we should first explain  how to call an actor method before explaining multiple returns.

Reply:
Also, I think we can just use the previous Counter actor, don't need to define a new one.